src/main/java/com/google/sps/models/PCAPAnalysis.java

```diff
@@ -0,0 +1,28 @@
+package com.google.sps.models;
+
+public class PCAPAnalysis {
+
+  private String id;
+  private String[] ips;
+
+  public PCAPAnalysis(String id, String[] ips) {
```

Comment:
Can you add a comment on what is the purpose of this class. What is it trying to achieve by encapsulating a id and set of IPs.

Reply:
Just a placeholder until I integrate w/ Eric.

Reply:
In that case add a comment and/or TODO saying it should be modified / removed after some integration.

Reply:
Changed.